README.md

```diff
@@ -4,9 +4,21 @@
 and systems administration tasks. It allows the user to run commands across a
 server farm.
 
-## Local usage
+## Local usage and environment setup
+
+The tool is intended to be configured and run on your local workstation/laptop. You should use `virtualenv` to create an isolated Python environment.
```

Comment:
This is rather strong for my liking. Could you either add a justification for saying "you should" use virtualenv, or say something like "The installation instructions have been tested on a XXX system:".

I don't use virtualenv, and get on just fine.